test/studies/bigint/bigintops.graph

```diff
@@ -0,0 +1,11 @@
+perfkeys: Elapsed 256 assign time =, Elapsed 1024 assign time =, Elapsed 256 on stmt time =
+files: bigintops.dat, bigintops.dat, bigintops.dat, bigintops.dat, bigintops.dat
+graphkeys: 256 bit addition, 1024 bit addition, 256 bit assign, 1024 bit assign, 256 bit pow
+ylabel: Time (seconds)
+graphtitle: BigInteger performance
+
+perfkeys: Elapsed 256 op time =, Elapsed 1024 op time =, Elapsed 1024 on stmt time =
+files: bigintops.dat
+graphkeys: 256 bit addition, 1024 bit addition, 1024 bit pow
+ylabel: Time (seconds)
+graphtitle: BigInteger 1024 bit pow performance
```

Comment:
the perfkeys here grabs "op time" which is not "pow", I think. But the title says "pow" performance"

Reply:
You're right, fixed these.